external_parser/event_processors/joined_event.h

```diff
@@ -300,6 +300,109 @@ struct ccb_joined_event : public typed_joined_event {
   }
 };
 
+// MOVE TO VW
+struct DecisionServiceInteraction_CA
```

Comment:
nit: either CamelCase or snake_case

Reply:
changed to DecisionServiceInteractionCats